modules/aleph/src/Aleph/AlephMaterial.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+use DateTime;
+use Drupal\aleph\Aleph\Handler\AlephMaterialHandler;
+
+/**
+ * Class AlephMaterial.
+ *
+ * @package Drupal\aleph\Aleph
+ */
+class AlephMaterial extends AlephMaterialHandler {
+
+  private $material;
+  private $loans;
+
+  /**
+   * AlephMaterial constructor.
+   *
+   * @param \Drupal\aleph\Aleph\AlephClient $client
+   *    Instance of the Aleph Client.
+   * @param \SimpleXMLElement $material
+   *    The SimpleXMLElement object for the material.
+   *
+   * @internal param $loans Array of loan objects from the Aleph API.*    Array of loan objects from the Aleph API.
+   */
+  public function __construct(AlephClient $client, \SimpleXMLElement $material) {
+    parent::__construct($client);
+    $this->material = $material;
```

Comment:
Don't do this. Pick out the interesting bits from `$material` and set them as properly documented properties. You can drop all the getters and simply make the properties public, or, if you insist, implement the getters as:

```php
public function getTitle() {
  return $this->title;
}
```

Then you'll have all the XML structure related code nicely together in the constructor, and the rest of the code will only have to deal with already sanitized data.